migrate/xlog/mpstream.c

```diff
@@ -276,7 +276,7 @@ mmpstream_encode_ext(struct mpstream *stream, const char *ext, uint32_t len,
 	assert(mp_sizeof_ext(len) < 5 + 1 + len);
 	char *data = mmpstream_reserve(stream, 1);
 	if (data) {
-		char *pos = mp_encode_ext(data, ext, len, type);
+		char *pos = mp_encode_ext(data, type, ext, len);
```

Comment:
Curious fact: The function was added to msgpuck in 2019, but this code is from 2016. The function is not used, though.

Reply:
LoL)